src/02_environment.jl

```diff
@@ -42,18 +42,10 @@ immutable Environment
         uuid = string(Base.Random.uuid4())
         timestamp = Libc.strftime("%Y-%m-%d %H:%M:%S", round(Int, time()))
         julia_sha1 = Base.GIT_VERSION_INFO.commit
-        package_sha1 = if isdir(".git")
-            sha1 = ""
-            try
-                sha1 = readchomp(`git rev-parse HEAD`)
-            end
-            if isempty(sha1)
```

Comment:
I don't think we can have an empty result; if `git rev-parse HEAD` doesn't spit something out, it throws an exception, so we continue along with the original value of `package_sha1`, which is `NULL`.